background.js

```diff
@@ -34,21 +56,35 @@ async function openTabInCurrentContainer() {
 }
 
 async function openContainerTab(contextNumber) {
-  let contexts;
-  try {
-    contexts = await getContexts();
-  } catch (e) {
-    console.error(e);
+  let context = await getContextFor(contextNumber);
+  console.log(context);
```

Comment:
Should this be console.debug()? I don't see console.log being used anywhere else in this repo.

Reply:
I didn't actually intend to be logging there, good catch!